docs/scarpet/api/Auxiliary.md

```diff
@@ -328,7 +329,12 @@ write_file('foo', 'shared_raw', 'five\n', 'six\n');
 
 read_file('foo', 'shared_text')     => ['one', 'two', 'three', '', 'four', '', 'five', 'six']
 </pre>
-  
+
+### `gamerule()`,`gamerule(rule)`,`gamerule(rule, value)`
+
+If run with no args, returns map of gamerules to their values.
+If run with just the rule name (lowercase, with underscores, e.g `do_mob_loot` instead of `doMobLoot`)
```

Comment:
since these names are used in vanilla, lets stick to camelCase here as well.

Reply:
ye Ill update when I migrate to other pr